src/apps/experimental/routes/user/display/index.tsx

```diff
@@ -0,0 +1,96 @@
+import Button from '@mui/material/Button';
+import { SelectChangeEvent } from '@mui/material/Select';
+import Stack from '@mui/material/Stack';
+import React, { useCallback } from 'react';
+
+import Page from 'components/Page';
+import globalize from 'scripts/globalize';
+import theme from 'themes/theme';
+import { DisplayPreferences } from './DisplayPreferences';
+import { ItemDetailPreferences } from './ItemDetailPreferences';
+import { LibraryPreferences } from './LibraryPreferences';
+import { LocalizationPreferences } from './LocalizationPreferences';
+import { NextUpPreferences } from './NextUpPreferences';
+import { useDisplaySettingForm } from './hooks/useDisplaySettingForm';
+import { DisplaySettingsValues } from './types';
+import LoadingComponent from 'components/loading/LoadingComponent';
+
+export default function UserDisplayPreferences() {
+    const {
+        loading,
+        submitChanges,
+        updateField,
+        values
+    } = useDisplaySettingForm();
+
+    const handleSubmitForm = useCallback((e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+        void submitChanges();
```

Comment:
Ah good question! It's usually a good practice to handle any promise response from functions either with an `await` or by returning the promise. If we're intentionally not waiting for a promise, we can use the `void` keyword. This also helps communicate intent to future devs by showing the promise response is intentionally ignored rather than a mistake. The project lint rules actually check for this and require that the promise is either awaited, returned, or marked with the `void` keyword.

As for sonarcloud, it seems like it's unable to tell that the `submitChanges` function is async. Maybe something to do with it being an async function wrapped in `useCallback`? I didn't have a chance to get into the nitty gritty of what sonarcube is actually seeing for this line.